src/plugins/core/image.ts

```diff
@@ -104,6 +112,25 @@ export class ImagePlugin extends CorePlugin<ImageState> implements ImageState {
     };
     this.dispatch("CREATE_FIGURE", { sheetId, figure });
   }
+
+  import(data: WorkbookData) {
+    for (const sheet of data.sheets) {
+      const images = (sheet.figures || []).filter((figure) => figure.tag === "image");
```

Comment:
:arrow_up: we could at the very least type FigureData.tag  :)

Reply:
> we could at the very least type FigureData.tag :)

I don't know. I'm not sure.
If we were not extensible I'd say 100% yes. But since anyone can add new figures...
I know we never cared about external typings up to know. Maybe we should since we are open-source and we want the types in odoo